Makefile

```diff
@@ -39,18 +39,12 @@ test:
 .PHONY: external-mypy
 external-mypy:
 	@echo "testing simple example with mypy (and python to check it's sane)..."
-	python tests/mypy_test_success.py
-	mypy tests/mypy_test_success.py
+	python tests/mypy/success.py
+	mypy tests/mypy/success.py
 	@echo "checking code with incorrect types fails..."
-	@mypy tests/mypy_test_fails1.py 1>/dev/null; \
+	@mypy tests/mypy/fail1.py 1>/dev/null; \
 	  test $$? -eq 1 || \
 	  (echo "mypy_test_fails1: mypy passed when it should have failed!"; exit 1)
-	@mypy tests/mypy_test_fails2.py 1>/dev/null; \
```

Comment:
I kept a single success and fail case just to ensure the `mypy.api.run` calls don't result in false negatives. I think it could make sense to just remove them all though if we are willing to trust in `mypy.api.run`.

Reply:
Nevermind, since you can't use `mypy.api.run` without `typing_extensions` installed, I kept these all.